test/python-bdk.spec.js

```diff
@@ -26,33 +30,73 @@ describe('Python BDK', () => {
         language: 'python',
         appId: 'app-id'
       }).then((dir) => {
+        assertCommonFilesGenerated(dir);
+      })
+  })
+
+  it('Generate 2.0 python ext-app', () => {
```

Comment:
I think for Java we are building the project part of the test, is that possible for Python too?
(the tricky thing would be the image we are using to run the tests, it has Java + Node, does it have Python...)

Reply:
Opened https://perzoinc.atlassian.net/browse/PLAT-10747 for improvements